backend/apps/slack/management/commands/slack_sync_data.py

```
@@ -41,90 +30,116 @@ def handle(self, *args, **options):
-                    self.style.ERROR(f"Failed to fetch members: {e.response['error']}")
+                if e.response.status_code == 429:
+                    retry_after = int(e.response.headers.get("Retry-After", 1))
+                    self.stdout.write(self.style.WARNING(f"Rate limited. Sleeping {retry_after}s..."))
+                    time.sleep(retry_after)
+                    retries += 1
+                else:
+                    raise e
+            retries += 1
+            backoff = 2 ** retries + secrets.randbelow(1000) / 1000
+            time.sleep(backoff)
+        raise SlackApiError("Max retries exceeded", {})
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)